eng/testing/tests.wasm.targets

```diff
@@ -9,11 +9,11 @@
     <!-- Some tests expect to load satellite assemblies by path, eg. System.Runtime.Loader.Tests,
          so, just setting it true by default -->
     <IncludeSatelliteAssembliesInVFS Condition="'$(IncludeSatelliteAssembliesInVFS)' == ''">true</IncludeSatelliteAssembliesInVFS>
+    <WasmEmitSymbolMap Condition="'$(WasmEmitSymbolMap)' == ''">true</WasmEmitSymbolMap>
 
     <!-- Run only if previous command succeeded -->
     <_ShellCommandSeparator Condition="'$(OS)' == 'Windows_NT'">&amp;&amp;</_ShellCommandSeparator>
     <_ShellCommandSeparator Condition="'$(OS)' != 'Windows_NT'">&amp;&amp;</_ShellCommandSeparator>
-    <WasmNativeStrip>false</WasmNativeStrip>
```

Comment:
are there any potential problems caused by disabling strip - for example, does this make our tests less representative because now they won't surface bugs that only occur when stripping?

Reply:
This is actually set earlier in the file, and it was redundant here, that's why I removed it. So, no effective change.

Reply:
re:disabling-strip, yeah, that itself could maybe not be completely representative. Once the symbolication work is fully merged (another PR needed), we can maybe drop this, so it is the same as user builds.